modules/storage-api/src/test/java/org/apache/ignite/internal/storage/util/MvPartitionStoragesTest.java

```diff
@@ -326,8 +326,6 @@ void testStartRebalance() {
 
         assertThrowsWithMessage(StorageRebalanceException.class, () -> startRebalanceMvStorage(0),
                 "Storage in the process of starting a rebalance");
-        assertThrowsWithMessage(StorageRebalanceException.class, () -> abortRebalanceMvStorage(0),
-                "Storage in the process of starting a rebalance");
```

Comment:
I do not think that this is necessary, since this cancellation will abort the previously pushed rebalance.
This behavior is being tested in the new test **testAbortRebalanceBeforeFinishStartRebalance**.

Reply:
Great!